### ISSUE TITLE
[Feature Request] Change Volume with Mouse Wheel Scroll

### ISSUE THREAD

**Issue by blaenk** [FrankerFaceZ/FrankerFaceZ#81]
It would be nice to be able to scroll the mouse wheel while hovering over the video in order to manipulate the video's volume, opt-in if need be.

The [YouTube Magic Actions](https://chrome.google.com/webstore/detail/magic-actions-for-youtube/abjcfabbhafbcdfjoecdgepllmpfceif?hl=en) extension does something like this for YouTube videos.

**Comment by SirStendec** [comment]
Added in 03c825c39835d2b3b10b157096eff550711db81b

**Comment by blaenk** [comment]
Amazing, thank you very much! I'm sure people will find it useful.

Out of curiosity, I haven't worked on chrome extensions myself before, but I figure there's some time before new versions are available for users, perhaps due to some vetting process or something. However, I noticed that the extension page [here](https://chrome.google.com/webstore/detail/frankerfacez/fadndhdgpmmaapbmfcknlfgcflmmmieb) shows that it was last updated in February 25, 2016, which made me wonder if I was on some older fork or something, but it's the version served by the frankerfacez site. I also noticed that it's at version 1.57 compared to 3.5.418 in that commit. Is the version in the commit for internal use or something?

Or is it the case that the extension in the chrome webstore doesn't need to be updated because the code changes are pulled in remotely or something?

Thanks again!
